mac

```diff
@@ -146,35 +148,13 @@ brew_install_or_upgrade 'imagemagick'
 brew_install_or_upgrade 'qt'
 brew_install_or_upgrade 'hub'
 brew_install_or_upgrade 'node'
-
-brew_install_or_upgrade 'rbenv'
-brew_install_or_upgrade 'ruby-build'
-
-# shellcheck disable=SC2016
-append_to_zshrc 'eval "$(rbenv init - --no-rehash zsh)"' 1
+brew_install_or_upgrade 'vim'
```

Comment:
I *think* that homebrew vim should override “system" vim automatically because of the `PATH` update on `ln. 126`. Homebrew hasn’t always had `vim`, but they’ve recently changed their minds on including it.

I’m totally open to MacVim, but this seemed like a nicer (one-liner) approach.